WebContent/js/components/JobFile.js

```diff
@@ -68,7 +68,8 @@ class JobFile extends React.Component {
 
     openInNewWindow() {
         const { job, file } = this.props;
-        const newWindow = window.open(`/#/viewer?jobName=${job.get('jobName')}&jobId=${job.get('jobId')}&fileId=${file.id}`, '_blank');
+        const newWindow = window.open(`${window.location.origin}${window.location.pathname}`
```

Comment:
Abstract out BASE_URI as a separate function, instead of computing inline
`window.open(`${window.location.origin}${window.location.pathname}`
may be in utilities/urlUtils.js

Reply:
I've extracted out to a variable to make the code easier to read but I don't want to push some string concatenation into a separate function that isn't used elsewhere.